### ISSUE TITLE
[Security] Cut stable release with upgraded lodash or cli-table3

### ISSUE THREAD

**Issue by mike-north** [ember-cli/ember-try#200]
I'm not sure whether 1.0 is nearly ready, but it would be nice to get a stable release cut. Until then, the most recent (0.2.23) will cause npm6 users to get warnings about this low-risk lodash security issue.

https://nodesecurity.io/advisories/577

It may be worth cutting an ember-try v0.3 or something.

Pinning to lodash >=4.17.5 may do the trick, assuming everything else is happy with that version.

**Comment by rwjblue** [comment]
Agreed. I think we can do both: release a patch version of 0.2 and release 1.0.0.
